### DISCUSSION TITLE
Table responsive stack doesn't work anymore

### DISCUSSION THREAD

**Discussion by Jo-Philips** [primefaces/community#2638] [PrimeNG]
In the current version the responsiveness (stack option) of the table component is broken. 
The columns do not change to rows anymore.

**Comment by qwadrox** [comment]
The reason it is not working in the example in the docs is the domSanitizer. I don't fully understand why because I have the same dependencies in my project as the example and in my project it adds these classes correctly.  
<img width="981" alt="image" src="https://github.com/user-attachments/assets/24ba10fd-41e6-4acb-8a20-64bee4060422">
